src/containers/GalacticPropertiesComboContainer.jsx

```diff
@@ -99,18 +93,20 @@ class GalacticPropertiesComboContainer extends React.PureComponent {
   };
 
   generateNavItems(properties, activeProperty) {
-    return properties.map(property => {
-      const { name, color } = property;
+    return properties.map((property, i) => {
+      const { name } = property;
+      const isActive = activeProperty === property;
       return {
         leftAvatar: (
           <span>
-            <ScatterPlot style={{ fill: color }} />
+            <ScatterPlot />
             <span className="screen-reader-only">{name}</span>
           </span>
         ),
         primaryText: name,
-        className: classnames(propertyItem, {
-          [linkActive]: activeProperty === property,
+        className: classnames(style.propertyItem, {
+          [style[`inactive${i}`]]: !isActive,
+          [style[`active${i}`]]: isActive,
```

Comment:
Please reimplement where use numbered `property-item` classes (e.g. `property-item-1', `property-item-2', `property-item-3' etc.) instead of numbered `inactive` and `active`.  And then nested within each `property-item-X` class you can `&.active`.  In other words, we want like a top level class tied top these color indices instead of a bunch of these index-specific classes floating around.

Reply:
reimplemented as requested and update the scss to match